webpack/components/AnsibleRoles/components/AnsibleRolesImportButton.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { importUrl } from '../AnsibleRolesPageHelpers';
+
+export const AnsibleRolesImportButton = () => {
+  let dropdownItems;
+
+  const {
+    response: { results },
+    status,
+  } = useAPI('get', '/api/v2/smart_proxies');
```

Comment:
Following up on https://github.com/theforeman/foreman_ansible/pull/696#discussion_r1636410033 I'd imagine this  should work:
```suggestion
  } = useAPI('get', '/api/v2/smart_proxies', { params: { feature: 'Ansible' }});
```

Reply:
Hmm, for me this still returns the `centos8-katello-devel-stable.example.com` Smart Proxy...
I don't want to rule out though that this is caused by environment being broken.

Reply:
Doesn't that proxy just have that feature? Could also be a bug in the API, in which case we'd need to fix that as well.

Reply:
I am not sure. Infrastructure > Smart Proxies doesn't say that the Smart Proxy supports the Ansible feature and neither does the UI, but maybe I am missing something...
I have committed your suggestion and commented out the client-side filtering for now and will re-test this on a fresh machine.